src/main/java/frc/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -45,7 +49,7 @@ public ElevatorSubsystem() {
 
         extensionMotor = new CANSparkMax(ElevatorConstants.EXTENSION_ID, MotorType.kBrushless);
         extensionMotor.setIdleMode(IdleMode.kBrake);
-        extensionMotor.setInverted(false);
+        extensionMotor.setInverted(true);
```

Comment:
be careful changing this, I think it should still be false

Reply:
Let's just test it sometime